components/sections/DescriptionSection.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useEffect, useRef } from "react";
+import { createRef } from "react";
+import Wrapper from "../wrappers/Wrapper";
+import Arrow from "../Arrow";
+import Button from "../ui/Button";
+
+function DescriptionSection() {
+  const desc = [
```

Comment:
It's better to rename this variable to a meaningful word and also try to add this moked data in your shared folder not here and then import it here